chamber/models/__init__.py

```diff
@@ -176,18 +208,17 @@ def _save(self, is_cleaned_pre_save=None, is_cleaned_post_save=None, force_inser
         )
 
         change = bool(self.pk)
-        changed_fields = set(self.changed_fields)
         kwargs.update(self._get_save_extra_kwargs())
 
-        self._pre_save(change, changed_fields, *args, **kwargs)
+        self._pre_save(change, self.changed_fields, *args, **kwargs)
```

Comment:
posílám to do pre_save a post_save vlastně jen kvůli zjednodušení a přehlednosti kódu. self.changed_fields je přeci jenom delší než changed_fields